### DISCUSSION TITLE
Is Flet production ready?

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [flet-dev/flet#1658] [General]
Hello,
Thanks for developing and maintaining Flet.
It seems that it is THE answer specially for PWA (as of now), for those who does not want to jail themselves in streamlit or ready made apps.

I am going to make an PWA MVP. and want to know if Flet is production ready?

**Comment by FeodorFitsner** [comment]
Thanks for looking at Flet as an option for your next project!

Could you please elaborate what do you include into definition of "production ready"?

**Comment by clstaudt** [comment]
One possible "definition" of "production-ready": There is already a big variety of real-world apps built with flet that new projects could look to for best practices and code examples. 

In that sense I believe the answer is no.

Another definition would be "at this point it provides enough to build and ship real-world apps". I think the answer is yes, but without reference projects so far, that is a bet and an experiment.

**Comment by Shahin-rmz** [comment]
Thanks for the replies.
The term *production ready* is somehow subjective as @clstaudt said.
I would say, a reasonable `user authentication ` , `multipage support` and also enough stability to be able to handle multiple users is enough to say it is *production ready*.
2 first features are already implemented I think. [authentication](https://flet.dev/blog/user-authentication/) and [multpage](https://github.com/flet-dev/flet/issues/76).
The only question is about stability. :)

**Reply by abcnow** [reply]
@Shahin-rmz it has been already 8 months since your comment.. can you share your experience in terms of stability now? Is anyone using Flet in production already?

**Reply by Shahin-rmz** [reply]
Tbh, I have never started that project though.

**Comment by ndonkoHenri** [comment]
This should be moved to discussions 💬

**Comment by goyalyashpal** [comment]
@ndonkoHenri do you have the permissions to move it there to discussions?
@FeodorFitsner or maybe you?
